src/app/(frontend)/trip-reports/page.tsx

```diff
@@ -1,12 +1,92 @@
-export default async function TripReportsPage() {
+import React, { Suspense } from 'react'
+import Link from 'next/link'
+
+import { getTripReports } from '@/queries/tripReports'
+import { TripReportsGrid } from './_components/TripReportsGrid'
+import { TripReportsGridFallback } from './_components/TripReportsGridFallback'
+
+export default async function TripReportsPage({
+  searchParams: SearchParamsPromise,
```

Comment:
I think the renaming here is unnecessary, also should use camelCase for variable names
```suggestion
  searchParams,
```

Reply:
Noted!